src/main/java/frc/robot/poseestimator/PoseEstimatorMath.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator;
+
+import edu.wpi.first.math.geometry.*;
+
+public class PoseEstimatorMath {
+
+	public static double getKalmanRatio(double odometryStandardDeviation, double visionStandardDeviation) {
+		return odometryStandardDeviation / (odometryStandardDeviation + visionStandardDeviation);
+	}
+
+	public static Rotation2d combineVisionHeadingToGyro(
+		Rotation2d visionEstimatedHeading,
+		Rotation2d sampleDifferenceFromHeading,
+		Rotation2d currentEstimatedHeading,
+		double gyroStandardDeviation,
+		double visionStandardDeviation
+	) {
+		double combinedStandardDeviation = getKalmanRatio(gyroStandardDeviation, visionStandardDeviation);
+		Rotation2d estimatedHeadingAtSampleTime = currentEstimatedHeading.plus(sampleDifferenceFromHeading);
+		Rotation2d differenceFromVisionAndEstimatedHeading = getShortestAngleDifference(visionEstimatedHeading, estimatedHeadingAtSampleTime);
+		Rotation2d scaledDifferenceToAddToEstimatedHeading = differenceFromVisionAndEstimatedHeading.times(combinedStandardDeviation);
+		return currentEstimatedHeading.plus(scaledDifferenceToAddToEstimatedHeading);
+	}
+
+	public static Rotation2d getShortestAngleDifference(Rotation2d angle1, Rotation2d angle2) {
```

Comment:
```suggestion
	public static Rotation2d getAngleDistance(Rotation2d angle1, Rotation2d angle2) {
```
distance = shortest angle difference, in the context of a circle.